### DISCUSSION TITLE
CVE Exploitability for RELEASE.2022-06-11T19-55-32Z

### DISCUSSION THREAD

**Discussion by tcalotta** [minio/minio#16535] [Q&A] [answered]
Hi everyone,

We're currently using minio RELEASE.2022-06-11T19-55-32Z which Trivy says has a number of CVEs. From my research, none of these seem relevant to minio:
| CVE | My findings | Relevant? |
|---|---|---|
| [CVE-2022-2509](https://nvd.nist.gov/vuln/detail/CVE-2022-2509) | No reference to use of ```gnutls_pkcs7_verify``` function found| No|
|[CVE-2020-35527](https://nvd.nist.gov/vuln/detail/CVE-2020-35527) | No reference to use of```ALTER TABLE``` command found| No|
|[CVE-2022-42898](https://nvd.nist.gov/vuln/detail/CVE-2022-42898) | No reference to krb5 found in minio repo although I did find [reference](https://github.com/minio/minio/commit/6f2b4675fa781a8497501e5ee38d99099cb72de3) to krb5 for the "HDFS gateway"|No?|
|[CVE-2022-37434](https://nvd.nist.gov/vuln/detail/CVE-2022-37434) | No reference to use of```inflateGetHeader``` function found|No|
|[CVE-2022-40303](https://nvd.nist.gov/vuln/detail/CVE-2022-40303) | No reference to use of```libxml2```found|No|
|[CVE-2022-40304](https://nvd.nist.gov/vuln/detail/CVE-2022-40304) | No reference to use of```libxml2```found|No|

Can someone confirm if my analysis of the exploitability of these CVEs is accurate?

Thanks in advance.

**Comment by klauspost** [comment, marked as answer]
Please use the latest release.

We only offer support of earlier versions as part of our [subscriptions](https://min.io/pricing).

**Reply by tcalotta** [reply]
> Please use the latest release.
> 
> We only offer support of earlier versions as part of our [subscriptions](https://min.io/pricing).

We would like to but we're blocked by breakage in later versions, as discussed in https://github.com/minio/minio/discussions/15967

**Reply by harshavardhana** [reply]
The CVEs mentioned here are just container CVEs MinIO does not depend on it, it gets wrongly flagged.

**Reply by tcalotta** [reply]
> The CVEs mentioned here are just container CVEs MinIO does not depend on it, it gets wrongly flagged.

Thank you for the confirmation. I'm appreciative of your time and input here.
